packages/sdk-browser/src/sdk-client.ts

```diff
@@ -27,7 +29,7 @@ import {
   StorageKeys,
   getStorageValue,
   removeStorageValue,
-  setStorageValue,
```

Comment:
part of me feels like we should still expose these helpers to manipulate storage, just in case people want to do that

Reply:
@andrewkmin that is still exposed its exported in the index.ts file